app/utils/estimate_prioritization/prioritization_criteria.py

```diff
@@ -7,8 +7,8 @@
     ],
     'adjustment_testadj': [
         lambda estimate: (estimate['pop_adj'] is True) and (estimate['test_adj'] is True),
-        lambda estimate: (pd.isna(estimate['pop_adj'])) and (estimate['test_adj'] is True),
         lambda estimate: (estimate['pop_adj'] is True) and (pd.isna(estimate['test_adj'])),
+        lambda estimate: (pd.isna(estimate['pop_adj'])) and (estimate['test_adj'] is True),
```

Comment:
Seems fair to me @123chrisc - could certainly make a cogent argument that now with higher seroprevalence and better assays, we should prioritize pop-adj test-unadj in these cases

Reply:
Variables were all checked to match. Sorting pooling functions now!

Reply:
All variables do have a pooling function. They are also matched to the airtable_fields_config.py